apps/client/src/components/VisualStyles.js

```diff
@@ -1,11 +1,12 @@
 import styled from "styled-components";
+import { Card } from "@datapunt/asc-ui";
 
 export const FigCaption = styled.figcaption`
   font-style: italic;
 `;
 
-export const Figure = styled.figure`
-  min-height: 1px; /* IE11 Bug */
+export const StyledCard = styled(Card)`
```

Comment:
You replaced the `figure` element but kept the `figcaption` element. This is bad practice, because the `figcaption` element can only be placed as the first or last child of the `figure` element.